web/modules/custom/dpl_reservations/src/Plugin/Block/ReservationListBlock.php

```diff
@@ -85,16 +89,18 @@ public function build(): array {
       'branches-config' => DplReactAppsController::buildBranchesJsonProp($this->branchRepository->getBranches()),
 
       // Url.
-      'ereolen-my-page-url' => $config->get('ereolen_my_page_url'),
-      'pause-reservation-info-url' => $generalSettings->get('pause_reservation_info_url'),
+      'ereolen-my-page-url' => dpl_react_apps_format_app_url($config->get('ereolen_my_page_url'), GeneralSettingsForm::EREOLEN_MY_PAGE_URL),
+      'pause-reservation-info-url' => dpl_react_apps_format_app_url($generalSettings->get('pause_reservation_info_url'), GeneralSettingsForm::PAUSE_RESERVATION_INFO_URL),
 
       // Config.
       'interest-periods-config' => DplReactAppsController::getInterestPeriods(),
-      'page-size-desktop' => $config->get('page_size_desktop'),
-      'page-size-mobile' => $config->get('page_size_mobile'),
-      'pause-reservation-start-date-config' => $generalSettings->get('pause_reservation_start_date_config') ?? '',
-      'reservation-detail-allow-remove-ready-reservations-config' => $generalSettings->get('reservation_detail_allow_remove_ready_reservations_config'),
-      'threshold-config' => $generalSettings->get('threshold_config'),
+      'page-size-desktop' => $config->get('page_size_desktop') ?? DplReservationsSettings::PAGE_SIZE_DESKTOP,
+      'page-size-mobile' => $config->get('page_size_mobile') ?? DplReservationsSettings::PAGE_SIZE_MOBILE,
+      'pause-reservation-start-date-config' => $generalSettings->get('pause_reservation_start_date_config') ?? GeneralSettingsForm::PAUSE_RESERVATION_START_DATE_CONFIG,
```

Comment:
Please remember to add undeclared dependencies here.

Reply:
👍